torchvision/prototype/datasets/video_utils.py

```diff
@@ -0,0 +1,178 @@
+import random
+from typing import Any, Dict, Iterator
+
+import av
+import numpy as np
+import torch
+from torchdata.datapipes.iter import IterDataPipe
+from torchvision.io import video, _video_opt
```

Comment:
I'm not sure if I would use `_video_opt` in here.

Reply:
Sure.
Any particular reason why not?